src/style.toit

```diff
@@ -304,15 +304,15 @@ class RoundedCornerBorder extends InvisibleBorder:
   - $corner-radius: The radius of the rounded corners.
   - $block: A block to call to draw each corner.
   Block arguments are:
-  - byte-opacity: a 64 entry byte array of 8x8 opacity values, or null if the patch is fully transparent.
-  - bit-opacity: an 8 entry byte array of 8x8 opacity values, or null if the patch is fully transparent.
+  - byte-opacity: a 256 entry byte array of 16x16 opacity values, or null if the patch is fully transparent.
+  - bit-opacity: an 32 entry byte array of 16x16 opacity values, or null if the patch is fully transparent.
   - x: the x coordinate of the top left corner of the patch.
   - y: the y coordinate of the top left corner of the patch.
   - orientation: the orientation to draw the patch, one of $ORIENTATION-0, $ORIENTATION-90, $ORIENTATION-180, $ORIENTATION-270.
   */
   draw-corners_ left/int top/int right/int bottom/int corner-radius/int [block]:
-    for j := 0; j < corner-radius; j += 8:
-      for i := 0; i < corner-radius; i += 8:
+    for j := 0; j < corner-radius; j += 16:
```

Comment:
You could use constants.
Would make it easier to tweak and give these constants meaning.